test/src/mesh_modifiers/BreakMeshByBlockManual_3Blocks.C

```diff
@@ -0,0 +1,196 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "BreakMeshByBlockManual_3Blocks.h"
+#include "MooseMesh.h"
+
+registerMooseObject("MooseApp", BreakMeshByBlockManual_3Blocks);
```

Comment:
Make sure you register this with the MooseTestApp, not MooseApp. You will then have to add a flag to your test spec `allow_test_objects = true`